capabilities.json

```diff
@@ -105,15 +105,18 @@
     "objects": {
         "labels": {
             "displayName": "Data labels",
+            "displayNameKey": "Visual_LabelsDataLabels",
```

Comment:
It'd be better to use `Visual_DataPointsLabels` instead.
Please try to find the same value in Power BI resource before including a new property